third_party/docfx/templates/devsite/UniversalReference.html.primary.tmpl

```diff
@@ -1,7 +1,10 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 {{!master(layout/_master.tmpl)}}
 
-<h1 class="page-title">{{#title}}{{title}}{{/title}}{{^title}}{{>partials/title}}{{/title}}</h1>
+<h1 class="page-title">
+{{#_displayFriendlyApiName}}

```

Comment:
I don't think we need `_displayFriendlyApiName`. Values are truthy in the template. So, we should just be able to do

```
{{#friendlyApiName}}
{{friendlyApiName}}
{{/friendlyApiName}}

Reply:
I was hoping to have a flag, ```_displayFriendlyApiName```, to either enable or disable this change from doc-pipeline.  Ruby libraries are already generating yaml with the ```friendlyApiName``` key so updating the template would effectively make this change live on prod for at least some Ruby libraries. However, I think just relying on the ```friendlyApiName``` key (and not changing the docfx JSON) is okay too as this is a pretty safe change. Wdyt?

Reply:
That's a good point, we can just use the ```friendlyApiName``` key directly then.